### ISSUE TITLE
NTP binding: unexpected log message regarding invalid date format

### ISSUE THREAD

**Issue by lolodomo** [eclipse/smarthome#3962]
When runnign the demo sitemap, I see this log:
```
Could not format ntp:ntp:demo with DateFormat yyyy-MM-dd HH:mm:ss z. Using default format.
```
`yyyy-MM-dd HH:mm:ss z` is the default value for the channel configuration parameter. It looks like it is not a valid format.

**Comment by lolodomo** [comment]
I fixed it in the PR with internationalization of the binding.

**Comment by lolodomo** [comment]
Has been fixed by merged PR #3961
